src/Jenssegers/Mongodb/Query/Builder.php

```diff
@@ -239,11 +239,21 @@ public function getFresh($columns = [])
             // Add grouping columns to the $group part of the aggregation pipeline.
             if ($this->groups) {
                 foreach ($this->groups as $column) {
-                    $group['_id'][$column] = '$' . $column;
```

Comment:
See docs, https://docs.mongodb.com/manual/reference/operator/aggregation/group/, param id is required